tests/zfs-tests/tests/functional/cli_root/zpool_scrub/setup.ksh

```diff
@@ -38,10 +38,5 @@ verify_disk_count "$DISKS" 2
 default_mirror_setup_noexit $DISK1 $DISK2
 
 mntpnt=$(get_prop mountpoint $TESTPOOL)
-typeset -i i=0
-while ((i < 10)); do
-	log_must $DD if=/dev/urandom of=$mntpnt/bigfile.$i bs=1024k count=100
-	((i += 1))
-done
-
+log_must $DD if=/dev/zero of=$mntpnt/bigfile bs=10M count=10
```

Comment:
Let's use `$FILE_WRITE` with a fill character so we don't deplete the entropy pool and the test runs quickly.